### PR TITLE
Update content of getting started

### PR DESCRIPTION
**Pull request by sebastiankb**
rendered version: https://cdn.statically.io/gh/w3c/wot-marketing/gettingStarted/docs/getting-started/index.html

### CODE REVIEW

**Comment by sebastiankb** [comment]
@mlagally provides some review comments for the current PR

File: docs/getting-started/index.html [hunk 1]
```diff
@@ -4,19 +4,55 @@
 group: "navigation"
 ---
 <div>
-	<h2>Getting Started</h2>
-	<p>Below you will find various materials to give you an introduction to the W3C Web of Things and its technology.</p>
+	<h2>Introduction</h2>
+	<p>The W3C Web of Things (WoT) seeks to counter the fragmentation of the IoT, making it much easier to create 
+		applications without the need to master the disparate variety of IoT technologies and standards.
```
Comment (mlagally) [inline docs/getting-started/index.html]: WoT defines an abstract architecture that is based on a set of requirements that were derived from use cases for multiple application domains. 
The architecture defines a basic conceptual framework that can be mapped onto a variety of concrete deployment scenarios and use cases.

Use cases and application domains include: 
- Consumer
- Industrial
- Transportation & Logistics
- Engineering and Construction
- Agriculture
- Healthcare
- Environment Monitoring
- Smart Cities
- Smart Buildings
- Connected Cars

File: docs/getting-started/index.html [hunk 2]
```diff
@@ -4,19 +4,55 @@
...
+		blocks is given here:</p>
+	<h3>Thing</h3>
+	<p>A Thing is an abstraction of a physical or virtual entity that provides interactions to and participates 
+		in the Web of Things. </p>
+	<h3>Interaction Affordances: Properties, Actions, Events</h3>
+	<p>The Interaction Model of W3C WoT introduces an intermediate abstraction that formalizes the mapping from application 
+		intent to concrete protocol operations and also narrows the possibilities how Interaction Affordances can be modeled.
+		So, Things may offer three other types of Interaction Affordances defined by this specification: 
+		Properties, Actions, and Events. 
+	</p>
```
Comment (mlagally) [inline docs/getting-started/index.html]: <h3>Consumer</h3>

A consumer of a thing is able to parse and process the Thing Description to be able to interact with the thing. This interaction can be over a direct connection or via one or more intermediaries.

<h3>Intermediary</h3>

Intermediaries can act as proxies for Things, where the Intermediary has a WoT Thing Description similar to the original Thing, but which points to the WoT Interface provided by the Intermediary. Intermediaries may also augment existing Things with additional capabilities or compose a new Thing out of multiple available Things, thereby forming a virtual entity. To Consumers, Intermediaries look like Things, as they possess WoT Thing Descriptions and provide a WoT Interface.

File: docs/getting-started/index.html [hunk 3]
```diff
@@ -4,19 +4,55 @@
 group: "navigation"
 ---
 <div>
-	<h2>Getting Started</h2>
-	<p>Below you will find various materials to give you an introduction to the W3C Web of Things and its technology.</p>
+	<h2>Introduction</h2>
+	<p>The W3C Web of Things (WoT) seeks to counter the fragmentation of the IoT, making it much easier to create 
+		applications without the need to master the disparate variety of IoT technologies and standards. 
+		To realize this, WoT specifies some principles and technology building blocks that can be used to
```
Comment (mlagally) [inline docs/getting-started/index.html]: To realize this, WoT specifies some principles and technology building blocks that were derived from requirements addressing these use cases. These building blocks can be used to

**Comment by sebastiankb** [comment]
thanks @mlagally. Your input is now included.

**Pull request merged**